src/Apps/Artist/Routes/AuctionResults/ArtistAuctionResults.tsx

```diff
@@ -1,193 +1,201 @@
-import { Col, Flex, Row } from "@artsy/palette"
+import { Col, Flex, Row, Sans, Separator, Serif } from "@artsy/palette"
 import { ArtistAuctionResults_artist } from "__generated__/ArtistAuctionResults_artist.graphql"
 import { PaginationFragmentContainer as Pagination } from "Components/v2/Pagination"
-import React, { Component } from "react"
+import React, { useState } from "react"
 import { createRefetchContainer, graphql, RelayRefetchProp } from "react-relay"
-import { Subscribe } from "unstated"
-import { ArtistAuctionDetailsModal } from "./ArtistAuctionDetailsModal"
+import useDeepCompareEffect from "use-deep-compare-effect"
 import { AuctionResultItemFragmentContainer as AuctionResultItem } from "./ArtistAuctionResultItem"
-import { AuctionResultsState } from "./state"
-// import { TableColumns } from "./TableColumns"
-// import { TableSidebar } from "./TableSidebar"
+import { TableSidebar } from "./TableSidebar"
 
 import { Box, Spacer } from "@artsy/palette"
 
-import { LoadingArea, LoadingAreaState } from "Components/v2/LoadingArea"
-import createLogger from "Utils/logger"
+import { LoadingArea } from "Components/v2/LoadingArea"
+import { isEqual } from "lodash"
+import { usePrevious } from "Utils/Hooks/usePrevious"
+import { ArtistAuctionDetailsModal } from "./ArtistAuctionDetailsModal"
+import {
+  AuctionResultsFilterContextProvider,
+  useAuctionResultsFilterContext,
+} from "./AuctionResultsFilterContext"
 import { AuctionResultsCountFragmentContainer as AuctionResultsCount } from "./Components/AuctionResultsCount"
 import { SortSelect } from "./Components/SortSelect"
 
-const logger = createLogger("ArtistAuctionResults.tsx")
+// const logger = createLogger("ArtistAuctionResults.tsx")
 
 const PAGE_SIZE = 10
 
 interface AuctionResultsProps {
   relay: RelayRefetchProp
   artist: ArtistAuctionResults_artist
-  sort: string
 }
 
-class AuctionResultsContainer extends Component<
-  AuctionResultsProps,
-  LoadingAreaState
-> {
-  state = {
-    isLoading: false,
-  }
+const AuctionResultsContainer: React.FC<AuctionResultsProps> = ({
+  artist,
+  relay,
+}) => {
+  const filterContext = useAuctionResultsFilterContext()
 
-  loadNext = () => {
-    const {
-      artist: {
-        auctionResultsConnection: {
-          pageInfo: { hasNextPage, endCursor },
-        },
-      },
-    } = this.props
+  const { sort } = filterContext.filters
+
+  const loadNext = () => {
+    const { hasNextPage, endCursor } = pageInfo
 
     if (hasNextPage) {
-      this.loadAfter(endCursor)
+      this.loadAfter(artist.slug, endCursor, relay, setIsLoading)
```

Comment:
With multiple things that need to be passed it's be a little more flexible if the function accepted an object rather than an explicitly ordered set of arguments, like: 

```tsx
loadAfter({ slug, endCursor, relay, setIsLoading }) 
```

But in this case, since everythign is already in the component function scope no need to pass anything. 

(Also note, no `this`)

Reply:
Ah good catch 🙌

Reply:
fixed in https://github.com/artsy/reaction/pull/3131/commits/3100aa594ca7b69ad2e40141c458a9c73f3fb017